containerfiles/unstable

```diff
@@ -0,0 +1,150 @@
+  vim \
+  webkit2gtk4.1 \
+  weston-session \
+  wget \
+  wireplumber \
+  xdg-utils \
+  xorg-x11-server-Xvfb \
+  xwininfo \
+  zenity
+
+# downgrade gamescope (temporary fix for Nvidia issues)
+RUN wget https://kojipkgs.fedoraproject.org/packages/gamescope/3.14.2/3.fc41/x86_64/gamescope-3.14.2-3.fc41.x86_64.rpm \
+ && ${CMD_INSTALL} gamescope*.rpm \
+ && rm gamescope*.rpm
+
+# uninstall default kernel
+RUN dnf remove -y kernel kernel-core kernel-devel kernel-devel-matched kernel-modules kernel-modules-core
+
+# install custom kernel
+RUN wget https://github.com/hhd-dev/kernel-bazzite/releases/download/${kver}/kernel-${kver}.bazzite.fc41.x86_64.rpm \
+ && wget https://github.com/hhd-dev/kernel-bazzite/releases/download/${kver}/kernel-core-${kver}.bazzite.fc41.x86_64.rpm \
+ && wget https://github.com/hhd-dev/kernel-bazzite/releases/download/${kver}/kernel-devel-${kver}.bazzite.fc41.x86_64.rpm \
+ && wget https://github.com/hhd-dev/kernel-bazzite/releases/download/${kver}/kernel-devel-matched-${kver}.bazzite.fc41.x86_64.rpm \
+ && wget https://github.com/hhd-dev/kernel-bazzite/releases/download/${kver}/kernel-modules-${kver}.bazzite.fc41.x86_64.rpm \
+ && wget https://github.com/hhd-dev/kernel-bazzite/releases/download/${kver}/kernel-modules-core-${kver}.bazzite.fc41.x86_64.rpm \
+ && wget https://github.com/hhd-dev/kernel-bazzite/releases/download/${kver}/kernel-modules-extra-${kver}.bazzite.fc41.x86_64.rpm \
+ && ${CMD_INSTALL} kernel*.rpm \
+ && rm kernel*.rpm
+
+# Install Nvidia driver
+RUN ${CMD_INSTALL} \
+  --setopt=disable_excludes=* \
+  kernel-srpm-macros
+
+RUN ${CMD_INSTALL} \
+  akmod-nvidia \
+  nvenc \
+  nvidia-driver \
+  nvidia-driver-NvFBCOpenGL \
+  nvidia-driver-libs.i686
+
+RUN mkdir -p /run/akmods; for k in $(ls -1 /usr/src/kernels); do \
+  akmods --force --kernels "${k}" --kmod "nvidia" || exit 1; \
+  ls /usr/lib/modules/"${k}"/extra/nvidia/nvidia.ko.xz || exit 1; done; rm -r -f /run/akmods
+
+# Copy/override files
+COPY rootfs/etc/hostname /etc/
+COPY rootfs/etc/bluetooth/main.conf /etc/bluetooth/
+COPY rootfs/usr/lib/os-release-playtron /usr/lib/
+COPY rootfs/usr/share/plymouth/themes/spinner/watermark.png /usr/share/plymouth/themes/spinner/
+COPY rootfs/usr/lib/bootc/kargs.d/50-playtron.toml /usr/lib/bootc/kargs.d/
+
+# Set SELinux to permissive mode
+RUN sed -i s'/SELINUX=.*/SELINUX=permissive/'g /etc/selinux/config
+
+# Ensure services are enabled according to presets
+RUN systemctl preset-all
+RUN systemctl --user --global preset-all
+
+# Set graphical boot target
+RUN cd /usr/lib/systemd/system && rm default.target && ln -s graphical.target default.target
+
+# Build initramfs
+RUN KERNEL=$(echo /lib/modules/*/vmlinuz | cut -d'/' -f4) \
+  && dracut --reproducible -v --omit-drivers nouveau --add 'ostree' -f --no-hostonly --kver ${KERNEL} /lib/modules/${KERNEL}/initramfs.img
+
+# Clear /var/run
+RUN rm -rf /var/run
+RUN ln -s /run /var/run
+
+# View final packages
+RUN rpm -qa | sort --ignore-case
```

Comment:
We can add one more command at the end of both Containerfiles to verify that everything is built according to bootc standards: `bootc container lint`